pkgs/applications/version-management/gex/default.nix

```diff
@@ -2,25 +2,32 @@
 , stdenv
 , rustPlatform
 , fetchFromGitHub
+, pkg-config
+, openssl
+# waiting on gex to update to libgit2-sys >= 0.15
+, libgit2_1_5
 }:
 
 rustPlatform.buildRustPackage rec {
   pname = "gex";
-  version = "0.3.3";
+  version = "0.3.8";
 
   src = fetchFromGitHub {
     owner = "Piturnah";
     repo = pname;
     rev = "v${version}";
-    hash = "sha256-oUcQKpZqqb8wZDpdFfpxLpwdfQlokJE5bsoPwxh+JMM=";
+    hash = "sha256-pjyS0H25wdcexpzZ2vVzGTwDPzyvA9PDgzz81yLGTOY=";
   };
 
-  cargoHash = "sha256-ZFrIlNysjlXI8n78N2Hkff6gAplipxSQXUWG8HJq8fs=";
+  nativeBuildInputs = [ pkg-config ];
+  buildInputs = [ openssl libgit2_1_5 ];
```

Comment:
Shouldn't libgit be in nativebuildinputs?

Reply:
How I understand it  relies on FFI, so we have to go this route.

https://discourse.nixos.org/t/failed-to-compile-openssl-sys-with-rust-on-macos/20785/2

https://johns.codes/blog/rust-enviorment-and-docker-build-with-nix-flakes

Reply:
I believe git is needed in nativeBuildInputs as it is shelled out to in the [build script](https://github.com/Piturnah/gex/blob/70d59626ae41e6cebb3eeb8b949d2e8ac8823f77/build.rs#LL8C21-L8C21)